sdks/python/src/opik/evaluation/metrics/llm_judges/g_eval/metric.py

```diff
@@ -134,6 +139,10 @@ def _parse_model_output(self, content: ModelResponse) -> score_result.ScoreResul
             if not (0.0 <= final_score <= 1.0):
                 raise ValueError
 
-            return score_result.ScoreResult(name=self.name, value=final_score)
+            # Get the reason
+            reason = json.loads(content.choices[0].message.content)["reason"]
```

Comment:
Just for better understanding, since the content contains multiple outputs (because of `top_logprobs`) which one is the reason related to? The first one? Maybe worth adding a comment because the understanding of this logic heavily depends on the understanding of model response.

Reply:
The way log_probs works is actually a bit different. Basically what happens is that the model generates one token at a time by choosing the token with the highest probability (more or less). When you ask for `log_probs`, you actually just get to see which tokens where not chosen for specific position.

So here the `reason` is fixed and relevant to the most likely token that was generated. I'll add comments here